tests/fixture-valid.js

```diff
@@ -611,6 +612,17 @@ export async function checkFixture(manufacturerKey, fixtureKey, fixtureJson, uni
           }
         }
 
+        /**
+         * Type-specific checks for StrobeSpeed capabilities.
+         */
+        function checkStrobeSpeedCapability() {
+          const otherCapabilityHasShutterStrobe = channel.capabilities.some(otherCapability => otherCapability.type === `ShutterStrobe`);
+          const hasOtherStrobeChannel = fixture.coarseChannels.some(otherChannel => otherChannel !== channel && otherChannel.type === `Strobe`);
```

Comment:
I'm not familiar enough with this program, so here's a dumb question: does this check all channels of the fixture, or just the channels used by the same mode (personality) as the current channel?

That is, is it possible for this to flag a false positive if `hasOtherStrobeChannel` finds a Strobe channel which is irrelevant because it's only used in some mode which never uses this `channel`?

Then again, if this only flagged one fixture in the whole database (which was actually an error), I suppose that is a crazy enough situation that no manufacturer actually does it.

Reply:
This checks all channels of the fixture, even those used only in different modes. I thought about writing a more elaborate check that tests each mode individually, but figured that it's probably not worth the effort for this rare situation.

When "`hasOtherStrobeChannel` finds a Strobe channel which is irrelevant because it's only used in some mode which never uses this `channel`", then no validation error is thrown (even if one should actually be thrown), so then the situation is basically the same as now without the check.